extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -60,23 +60,44 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
     supportsReject,
     staticWado,
     singlepart,
+    omitQuotationForMultipartRequest,
+    acceptHeader,
+    requestTransferSyntaxUID,
   } = dicomWebConfig;
 
   const dicomWebConfigCopy = JSON.parse(JSON.stringify(dicomWebConfig));
 
+  //Generate accept header depending on config params
+  let formattedAcceptHeader = utils.generateAcceptHeader(
+    acceptHeader,
+    requestTransferSyntaxUID,
+    omitQuotationForMultipartRequest
+  );
+
+  const authHeaders = userAuthenticationService.getAuthorizationHeader();
```

Comment:
You can't add the authorization header and then forget about it - the problem is that it gets updated/changed as it gets renewed, so you need to fetch it at the time of use.  I would suggest creating a method that generates the full set of headers/updates them as required.

Reply:
OK I got it that's why the get authorization header was duplicated, will fix it